app/models/spree/variant_decorator.rb

```diff
@@ -128,4 +128,21 @@ def destruction
       end
     end
   end
+
+  # We override this method for users to enter prices with dot or comma indifferently
+  # In Spree 2.3 it will be moved to Spree::LocalizedNumber.parse
+  # See https://github.com/spree/spree/commit/b817dab4a95c1beda3f18e42f0efcd2b7e64312a
+  def parse_price(price)
+    return price.to_d unless price.is_a?(String)
+
+    price = price.gsub(/[^\d.,]/, '')  # Replace all Currency Symbols, Letters and -- from the string
+    if price =~ /^.*[\.,]\d{1}$/       # If string ends in a single digit (e.g. ,2)
+      price += "0"                     # make it ,20 in order for the result to be in "cents"
+    end
+    unless price =~ /^.*[\.,]\d{2}$/   # If does not end in ,00 / .00 then
+      price += "00"                    # add trailing 00 to turn it into cents
+    end
+    price = price.gsub(/[\.,]/, '')    # Replace all (.) and (,) so the string result becomes in "cents"
+    price.to_d / 100                   # Let to_decimal do the rest
+  end
```

Comment:
I'd rather copy the whole `Spree::LocalizedNumber` class instead of copying the method two times. The problem I see is that we will eventually fix something in `variant_decorator.rb` but we'll forgot to also apply it in the `price_decorator` and viceversa.

Reply:
I thought of that, but I wasn't sure if it was a great idea to get ahead of the Spree upgrade. When doing the upgrade we'll probably want to remove what we added that become unnecessary (calls to `Spree::LocalizedNumber.parse`, use of `require`). Do you think it would be simpler?

Reply:
> we'll probably want to remove what we added that become unnecessary

Reply:
Fix'd